src/openrct2/Game.cpp

```diff
@@ -427,6 +427,20 @@ void game_convert_strings_to_rct2(rct_s6_data* s6)
     }
 }
 
+static void RemoveBadRideTypes()
+{
+    for (ride_id_t i = 0; i < RIDE_ID_NULL; ++i)
+    {
+        auto* ride = get_ride(i);
+        if (ride == nullptr || ride->type < RIDE_TYPE_COUNT)
+        {
+            continue;
+        }
+        reset_sprite_spatial_index();
+        ride_action_modify(ride, RIDE_MODIFY_DEMOLISH, GAME_COMMAND_FLAG_APPLY);
```

Comment:
This will network the action, is this supposed to happen? Its better to add GAME_COMMAND_FLAG_NO_SPEND otherwise its networked/queued and has side effects money wise.

Reply:
Hmm. If the network versions don't match, you can't connect anyway, right? But it sounds like the fix could be applied twice in such a case?

Reply:
I'm thinking of just making it so that you can't open the parks instead.

Reply:
My two cents would be to just not load the park and tell the user they may need to update OpenRCT2 in order to load the park.

Reply:
I'm with @Broxzier , it's a little nudge to use something newer